eo-runtime/pom.xml

```diff
@@ -34,11 +34,17 @@ SOFTWARE.
   <name>eo-runtime</name>
   <description>EO runtime library</description>
   <dependencies>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.8.1</version>
+      <scope>compile</scope>
```

Comment:
@marceloamadeu not a good idea :) Just move the class to `src/test` and you won't need `compile` scope here

Reply:
Done...